### PR TITLE
Add TikTok documents

### PR DESCRIPTION
**Pull request by AdrienFines**
TikTok has many documents, we shall discuss whether we want to include all of them or not :)

### CODE REVIEW

File: services/TikTok.json [hunk 1]
```diff
@@ -14,6 +14,33 @@
...
+      "contentSelector": "article"
+    },
+    "virtualItemsPolicy": {
+      "location": "https://www.tiktok.com/legal/virtual-items?lang=en",
+      "contentSelector": "article",
+      "filters": [
+        "selectOnlyEuropeanVirtualItemsPolicy"
+      ]
+    },
+    "openSourceUse": {
```
Comment (MattiSG) [inline services/TikTok.json:37]: This is just a list of licenses for open-source software used in TikTok. I don't see the point in tracking those in CGUs 🙂

File: services/TikTok.json [hunk 2]
```diff
@@ -14,6 +14,33 @@
...
+    },
+    "lawEnforcementGuidelines": {
+      "location": "https://www.tiktok.com/legal/law-enforcement?lang=en",
+      "contentSelector": "article"
+    },
+    "copyrightPolicy": {
+      "location": "https://www.tiktok.com/legal/copyright-policy?lang=en",
+      "contentSelector": "article"
+    },
+    "virtualItemsPolicy": {
```
Comment (MattiSG) [inline services/TikTok.json:30]: Let's call this “in-app purchases policy”.

File: services/TikTok.json (hunk 1 above)
Comment (AdrienFines) [reply services/TikTok.json]: Removed !

**Comment by AdrienFines** [comment]
I'm currently rebasing to master (at least trying...)

File: services/TikTok.json [hunk 3]
```diff
@@ -14,6 +14,29 @@
         "startBefore": "#privacy-eea",
         "endBefore": "#privacy-row"
       }
+    },
+    "communityGuidelines": {
+      "fetch": "https://www.tiktok.com/community-guidelines?lang=en",
+      "select": "article"
```
Comment (Ndpnt) [inline services/TikTok.json:20]: Use `.base-layout-container` for `select` to also select the document title "Community Guidelines"

**Comment by AdrienFines** [comment]
Should be ready

**Comment by Ndpnt** [comment]
Can you please squash all your commits in one? :)

**Comment by AdrienFines** [comment]
I hope it is ok now :) I really had an hard time managing branchs and rebase, maybe we can discuss it tomorrow if you have time !

**Pull request merged**